packages/node/src/index.ts

```diff
@@ -44,9 +44,8 @@ export {
 
 export { NodeClient } from './client';
 export { makeNodeTransport } from './transports';
-export { defaultIntegrations, init, lastEventId, flush, close, getSentryRelease } from './sdk';
+export { defaultIntegrations, init, defaultStackParser, lastEventId, flush, close, getSentryRelease } from './sdk';
```

Comment:
Why not export `nodeStackLineParser` at the top level?

Reply:
If we export from the top level and then import from there, will we not hit issues with bundlers trying to target the browser?

I just have painful memories of bundlers pulling in everything and erroring before they've even had a chance to do the treeshaking magic and remove the non-compatible code.

Reply:
This is a fair point - maybe we just stick with this for now then.